Formula/gismo.rb

```diff
@@ -0,0 +1,68 @@
+class Gismo < Formula
+  desc "C++ library for isogeometric analysis (IGA)"
+  homepage "https://gismo.github.io"
+  url "https://github.com/gismo/gismo/archive/refs/tags/v21.12.0.tar.gz"
+  sha256 "4001b4c49661ca8b71baf915e773341e115d154077bef218433a3c1d72ee4f0c"
+  license "MPL-2.0"
+
+  head do
+    url "https://github.com/gismo/gismo.git", branch: "stable"
+  end
+
+  depends_on "cmake" => :build
+  depends_on "openblas"
```

Comment:
Can it use Accellerate on macOS?

Reply:
Yes, I just tried and it works. What shall I do? At [Formula Cookbook -> linear algebra libraries](https://docs.brew.sh/Formula-Cookbook#linear-algebra-libraries) I read that 

> Packages requiring BLAS/LAPACK linear algebra interfaces should link to [OpenBLAS](https://www.openblas.net/) by adding depends_on "openblas" and (if built with CMake) passing -DBLA_VENDOR=OpenBLAS to CMake, rather than Apple’s Accelerate framework or the default reference lapack implementation. Apple’s implementation of BLAS/LAPACK is outdated and may introduce hard-to-debug problems. The reference lapack formula is fine, although it is not actively maintained or tuned.

Reply:
That isn't true anymore. Apple updated its implementation a couple releases ago.